android-package/build.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
+    repositories {
+        google()
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:4.1.2"
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven-publish'
+
+group = projectGroup
+version = projectVersion
+
+afterEvaluate {
+    publishing {
+        publications {
+            mavenRelease(MavenPublication) {
+                from components.release
+            }
+        }
+        /* publish to local build/repo folder */
+        repositories {
+            maven {
+                url "$buildDir/repo"
+            }
+        }
+    }
+}
+
+android {
+    compileSdkVersion 30
+}
+
+dependencies {
+    api ('org.brainflow:brainflow:0.0.1-SNAPSHOT') {
```

Comment:
in CI pipelines we change version by regexp/string replace (I dont like regexp, but maven can not read from env variables)

Reply:
> And I would like to check if its possible to generate aar wo gradle using maven only, and looks like its possible https://stackoverflow.com/questions/25077131/create-android-archive-libraryaar-with-maven-only I would like to use single build system if it works

It's possible to do this but takes some working with the maven internals that I haven't figured out yet.  I actually talked with a jna developer and they seemed to imply that using gradle would be the most stable, because google wasn't maintaining the maven path: https://groups.google.com/g/jna-users/c/QUFT16yJyt0/m/gapq1WL9AQAJ

Reply:
something maven apparently can read from is -D flags passed on the command line, it seems; not sure if it is easy to use that